### PR TITLE
[C-2902] Update the upload forms to use the typography component

### PR DESCRIPTION
**Pull request by Kyle-Shanks**
### Description
Update the text in the upload forms to use the typography component

### Dragons

N/A

### How Has This Been Tested?

Manaully tested

### How will this change be monitored?

N/A

### Feature Flags ###

N/A

### CODE REVIEW

File: packages/web/src/pages/upload-page/fields/SourceFilesView.tsx [hunk 1]
```diff
@@ -165,7 +166,9 @@ const StemListItem = ({
           textClassName={styles.dropdownText}
         />
       </div>
-      <div className={styles.title}>{metadata.title}</div>
+      <Text size='Small' strength='Strong'>
```
Comment (amendelsohn) [inline packages/web/src/pages/upload-page/fields/SourceFilesView.tsx:169]: do these need to be capitalized? idk why but I assumed we'd use lower

File: packages/web/src/pages/upload-page/fields/SwitchRowField.tsx [hunk 2]
```diff
@@ -34,8 +35,10 @@ export const SwitchRowField = (props: ToggleFieldProps) => {
   return (
     <div className={styles.root}>
       <div className={styles.content}>
-        <h3 className={cn(styles.title, styles.modalHeading)}>{header}</h3>
-        <p className={styles.description}>{description}</p>
+        <Text className={styles.title} variant='title' size='Large'>
```
Comment (amendelsohn) [inline packages/web/src/pages/upload-page/fields/SwitchRowField.tsx:38]: strange to me to have the variant lower and the size capitalized. Can we make them all accept lower?

File: packages/web/src/pages/upload-page/forms/AttributionModalForm.tsx [hunk 3]
```diff
@@ -227,9 +226,9 @@ const AttributionModalFields = () => {
       </SwitchRowField>
       <Divider />
       <div className={cn(layoutStyles.col, layoutStyles.gap4)}>
-        <div
-          className={cn(typeStyles.title, typeStyles.titleLarge)}
-        >{`${messages.isrc.header} / ${messages.iswc.header}`}</div>
+        <Text variant='title' size='Large'>
```
Comment (amendelsohn) [inline packages/web/src/pages/upload-page/forms/AttributionModalForm.tsx:229]: so much cleaner! I love it

**Review by amendelsohn — APPROVED** [review]
This is so nice! Great job on the component 🎉

File: packages/web/src/pages/upload-page/fields/SourceFilesView.tsx (hunk 1 above)
Comment (Kyle-Shanks) [reply packages/web/src/pages/upload-page/fields/SourceFilesView.tsx:169]: yea Randy, brought this up too. I have an update to make the all lower coming next 👍

**Review by dylanjeffers — APPROVED** [review]
Hell yeah

**Pull request merged**